tslint.json

```diff
@@ -55,7 +55,7 @@
       "ignore-params"
     ],
     "no-misused-new": true,
-    "no-non-null-assertion": true,
+    "no-non-null-assertion": false,
```

Comment:
I picked the default tslint that is generated by angular-cli, so it should be a quite good default.

Here about "no non-null assertion", I prefer to turn it off because there's often cases where a nullable type is known to be non-null because of a "business" relation, for instance in NumericBucketPoint,  "avg" , "min", "max" are nullable but known to be non-null when "empty" is false. That tslint option was returning many warnings because of that.